libs/search-widget/vite.config.js

```diff
@@ -3,33 +3,32 @@ import { svelte } from '@sveltejs/vite-plugin-svelte';
 import * as path from 'path';
 const sveltePreprocess = require('svelte-preprocess');
 
+const widgetFolder = process.argv[5] || 'widget';
+
 // https://vitejs.dev/config/
 export default defineConfig({
   resolve: {
     alias: [{ find: '@nuclia/core', replacement: path.resolve(__dirname, '../sdk-core/src/index.ts') }],
   },
   build: {
-    outDir: 'dist/libs/search-widget',
+    outDir: `dist/libs/search-${widgetFolder}`,
```

Comment:
It is a bit weird to hard-code search here, in the future we will probably have a `viewer-widget`, or an `annotation-widget`.
I would go with

```
outDir: `dist/libs/${widgetFolder}-widget`,
```
(and rename the 2 existing widgets folders into `search` and `search-video`)

or  
```
outDir: `dist/libs/${widgetFolder}`,
```
(and rename the 2 existing widgets folders into `search-widget` and `search-video-widget`)

Reply:
you're right, I'll do the change